src/Controller/LoginController.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+namespace App\Controller;
+
+use App\Model\UserManager;
+
+class LoginController extends AbstractController
+{
+    public function login(): string
+    {
+        $errors = [];
+        $connexion = [];
```

Comment:
![can-you-speak-english-sonic](https://user-images.githubusercontent.com/79161810/167582285-bd16b430-0c28-4781-94da-e723b0e91635.gif)

Reply:
et tu peux faire de manière plus concise `$errors = $connection = [];`